recipe/conda_build_config.yaml

```diff
@@ -17,7 +17,7 @@ m2w64_c_stdlib:                # [win]
 c_stdlib_version:              # [unix]
   - 2.17                       # [linux]
   - 2.17                       # [linux and os.environ.get("CF_CUDA_ENABLED", "False") == "True"]
-  - 2.17                       # [linux and os.environ.get("CF_CUDA_ENABLED", "False") == "True"]
+  - 2.28                       # [linux and os.environ.get("CF_CUDA_ENABLED", "False") == "True"]
```

Comment:
The CUDA compiler version shouldn't have anything to do with the sysroot we target. Needing newer images to run some of the existing CUDA 12 build infrastructure is solved by bumping the images, not the sysroot.

Reply:
Is this the migrator you have in mind? 

I feel like it cna likely be simplified now:

https://github.com/conda-forge/cupy-feedstock/blob/main/.ci_support/migrations/cuda12x.yaml